src/CloseButton.js

```diff
@@ -0,0 +1,10 @@
+function CloseButton({removeItem}) {
+    return (
+        <span
```

Comment:
again, this behave as button, but right now it's a span, which is wrong, please change it to proper HTML element

Reply:
I changed it to button.